### PR TITLE
Make README more verbose about contributions

### PR DESCRIPTION
**Pull request by breeswish**
Fix https://github.com/pingcap/tikv/issues/3150

### CODE REVIEW

**Comment by breeswish** [comment]
@queenypingcap @Hoverbear  PTAL

File: README.md [hunk 1]
```diff
@@ -140,8 +140,22 @@ Read our configuration guide to learn about the various [configuration options](
...
 
+In addition, TiKV team actively develops and maintains a bunch of dependencies used in TiKV, which you may also interested in:
+
+- [rust-prometheus](https://github.com/pingcap/rust-prometheus): The Prometheus client for Rust, our metric collecting and reporting library
+
+- [rust-rocksdb](https://github.com/pingcap/rust-rocksdb): Our RocksDB binding and wrapper for Rust
+
+- [raft-rs](https://github.com/pingcap/raft-rs): The Raft distributed consensus algorithm implemented in Rust
+
+- [grpc-rs](https://github.com/pingcap/grpc-rs): The gRPC library for Rust built on C Core library and futures
```
Comment (siddontang) [inline README.md]: built on gRPC C Core library and Rust Futures

File: README.md [hunk 2]
```diff
@@ -140,8 +140,22 @@ Read our configuration guide to learn about the various [configuration options](
 
 ## Contributing
 
+Contributions are welcome! For beginners, we have prepared many suitable tasks for you. Please checkout our [Help Wanted issues](https://github.com/pingcap/tikv/issues?q=is%3Aissue+is%3Aopen+label%3A%22S%3A+HelpWanted%22) for a list, in which we have also marked the difficulty level we thought. If you are planning something big, for example, relates to multiple components or changes current behaviors, make sure to open an issue to discuss with us before going on :)
+
 See [CONTRIBUTING](./CONTRIBUTING.md) for details on submitting patches and the contribution workflow.
 
+In addition, TiKV team actively develops and maintains a bunch of dependencies used in TiKV, which you may also interested in:
```
Comment (siddontang) [inline README.md]: be interested

**Comment by queenypingcap** [comment]
@lilin90 PTAL. Also please add the newly added document links to the README as well

**Comment by breeswish** [comment]
@queenypingcap Sorry but I can't catch it.. what's the "document links" you are referring to?

**Comment by breeswish** [comment]
@lilin90 I think your documentation link is broken

**Comment by lilin90** [comment]
@breeswish Updated now

**Comment by breeswish** [comment]
@lilin90 Also I think @queenypingcap might be talking about copying this new README into your documentation instead of adding a documentation link into this README?

**Comment by queenypingcap** [comment]
@breeswish Sorry for being unclear. I meant what @lilin90 did: add the TiKV docs links to TiKV wiki, and link them here.

**Comment by breeswish** [comment]
@queenypingcap I moved the documentation link to our "Using TiKV" section, PTAL

**Comment by queenypingcap** [comment]
LGTM.

**Pull request merged**